### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by WeitaiKang** [nickgkan/beauty_detr#48]
Hi authors, I found that in your loss function [here](https://github.com/nickgkan/butd_detr/blob/10570e0b6826d4a236b18c2c8fac5903866e1c60/models/losses.py#L532), you sum up all the number of ground truth bboxes among all the gpus, without averaging this number by the number of gpus. It seems that it would lower down the loss value if we use more gpus to train right? Is it a right way to do this?

**Comment by WeitaiKang** [comment]
I make a mistake. It is the issue about the code (https://github.com/yanmin-wu/EDA/blob/master/models/losses.py), which is based on this code, but they remove this [line](https://github.com/nickgkan/butd_detr/blob/10570e0b6826d4a236b18c2c8fac5903866e1c60/models/losses.py#L534)
